platformio.ini

```diff
@@ -728,6 +727,12 @@ platform = ${common_LPC.platform}
 extends  = common_LPC
 board    = nxp_lpc1769
 
+# BTT SKR 1.4 needs a UART3 tweak
+[env:LPC1768_btt_skr_v1_4]
+platform    = ${LPC1768.platform}
+extends     = env:LPC1768
+build_flags = ${env:LPC1768.build_flags} -DLPC_PINCFG_UART3_P4_28
+
```

Comment:
Should another environment be added for the SKR 1.4 Turbo/LPC1769? The only difference between the two is the MCU.